# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1437700393 14400
#      Thu Jul 23 21:13:13 2015 -0400
# Node ID 1625e6bd5ce7354ab70eb85d19d3d1d4927d0df2
# Parent  3e9bfbf6ae086aedd28b0f64e6d9acb2dfee2509
build: Move nonexistent dummy source files into lib directories

* libgui/module.mk (nodist_EXTRA_libgui_liboctgui_la_SOURCES): Rename file to
libgui/dummy.cc.
* liboctave/module.mk (nodist_EXTRA_liboctave_liboctave_la_SOURCES): Rename
file to liboctave/dummy.cc.

diff --git a/libgui/module.mk b/libgui/module.mk
--- a/libgui/module.mk
+++ b/libgui/module.mk
@@ -36,17 +36,17 @@ include libgui/src/module.mk
 include libgui/graphics/module.mk
 include libgui/qterminal-module.mk
 
 ## liboctgui merely collects a bunch of compiled convenience libraries.
 ## It has no source code itself.
 libgui_liboctgui_la_SOURCES =
 
 # Dummy C++ source to force C++ linking.
-nodist_EXTRA_libgui_liboctgui_la_SOURCES = dummy.cc
+nodist_EXTRA_libgui_liboctgui_la_SOURCES = libgui/dummy.cc
 
 libgui_liboctgui_la_LIBADD = \
   libgui/qterminal/libqterminal.la \
   libgui/src/libgui-src.la \
   libgui/graphics/libgui-graphics.la \
   $(top_builddir)/libinterp/liboctinterp.la \
   $(top_builddir)/liboctave/liboctave.la \
   $(LIBOCTGUI_LINK_DEPS)
diff --git a/liboctave/module.mk b/liboctave/module.mk
--- a/liboctave/module.mk
+++ b/liboctave/module.mk
@@ -61,17 +61,17 @@ include liboctave/operators/module.mk
 include liboctave/system/module.mk
 include liboctave/util/module.mk
 
 ## liboctave merely collects a bunch of compiled convenience libraries.
 ## It has no source code itself.
 liboctave_liboctave_la_SOURCES =
 
 # Dummy C++ source to force C++ linking.
-nodist_EXTRA_liboctave_liboctave_la_SOURCES = dummy.cc
+nodist_EXTRA_liboctave_liboctave_la_SOURCES = liboctave/dummy.cc
 
 liboctave_liboctave_la_LIBADD += \
   $(top_builddir)/libgnu/libgnu.la \
   $(LIBOCTAVE_LINK_DEPS)
 
 # Increment these as needed and according to the rules in the libtool manual:
 liboctave_liboctave_current = 3
 liboctave_liboctave_revision = 0
